src/oc/web/components/search.cljs

```diff
@@ -70,14 +73,17 @@
                                   s)}
   [s]
   (let [search-results (drv/react s store/search-key)
-        search-active? (drv/react s store/search-active?)]
+        search-active? (drv/react s store/search-active?)
+        result-count (if (< store/search-limit (:count search-results))
+                       store/search-limit
+                       (:count search-results))]
```

Comment:
@thepug Why do you repeat this result-count calculation? can you pass it to the `results-header` component to avoid recalc it there too?

Reply:
that would just make sense :)